src/core/redux/selectors/selectPropertyTypeIDsByFQN.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * @flow
+ */
+
+import _isArray from 'lodash/isArray';
+import {
+  Map,
+  Set,
+  getIn,
+  isCollection,
+} from 'immutable';
+import { Models } from 'lattice';
+import type { PropertyType, UUID } from 'lattice';
+
+import { EDM, PROPERTY_TYPES, PROPERTY_TYPES_INDEX_MAP } from '../constants';
+
+const { FQN } = Models;
+
+const EMPTY_MAP = Map();
+
+export default function selectPropertyTypeIDsByFQN(fqns :Set<FQN | string> | Array<FQN | string>) {
```

Comment:
1. this looks like a copy of `selectPropertyTypes` in `lattice-utils`
2. you can simplify this significantly:

```
import { ReduxUtils } from 'lattice-utils';

const { selectPropertyTypes } = ReduxUtils;

export default function select(fqns) {
  return (state) => {
    const propertyTypes :Map<UUID, PropertyType> = selectPropertyTypes(fqns)(state);
    return propertyTypes.map((propertyType) => propertyType.type).flip();
  };
}
```

Reply:
calling map(...).flip() would incur two extra intermediate collections as overhead, would it not?